packages/@webex/http-core/src/request/utils.ts

```diff
@@ -56,8 +56,23 @@ export async function prepareFetchOptions(options: any): Promise<any> {
 
   options.headers = options.headers || {};
 
+  if (options.json === true) {
+    // don't override existing accept header declared by user
+    options.headers.accept = options.headers.accept || options.headers.Accept || 'application/json';
+
+    // don't override existing content-type header declared by user
+    if (options.method !== 'GET' && options.method !== 'HEAD') {
```

Comment:
i am not sure about the content type defaulted to application/json

does browser handle it by * , I would check with tim once on this PR